pkg/domain/context.go

```diff
@@ -46,10 +46,10 @@ func (c *Context) Run() error {
 	return nil
 }
 
-func (c *Context) selectContext() (string, error) {
+func (c *Context) selectContext(currentContext string) (string, error) {
```

Comment:
you may not need this param as you pass in c.Config.CurrentContext, and this func already has access to the c object

Reply:
yeah was thinking that... but as this is only ever used it seemed silly to copy it

Reply:
there is a better way though

Reply:
nice one @macox, I've refactored based off your comment.